.travis.yml

```diff
@@ -2,12 +2,15 @@ language: r
 sudo: false
 cache: packages
 
+os:
+  - linux 
+  - osx 
+
```

Comment:
I don’t think this is necessary. I develop on macOS, and of course Neuroconductor tests there, so I think Linux and Windows are sufficient for the package’s mainline CI.